<?xml version="1.0"?>
<launch>

  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="1280" />
    <param name="image_height" value="720" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>

  <node pkg="verilook_ros" type="face_recognition_verilook_node" name="face_recognition_verilook_node" output="screen"> <!-- launch-prefix="xterm -e gdb -args"-->
    <param name="enable_database" value="true" type="bool"/>
    <param name="database_path" value="faces.db" type="str"/>
    <remap from="/face_recognition_verilook_node/event_out" to="remapped_output"/>
  </node>

</launch>

